mtags/src/main/scala/scala/meta/internal/metals/PackageIndex.scala

```diff
@@ -191,7 +191,7 @@ object PackageIndex {
           Classpath(v).entries
       }.toList
       entry <- entries
-      if entry.isFile
+      if entry.isFile && !entry.toNIO.endsWith("jfr.jar")
```

Comment:
what's `jfr.jar`? Maybe this is worth a comment?

Reply:
Good point, added (but not pushed yet).